web/api/v1/api.go

```diff
@@ -1207,10 +1204,7 @@ func (api *API) remoteRead(w http.ResponseWriter, r *http.Request) {
 		for i, query := range req.Queries {
 			err := api.remoteReadQuery(ctx, query, externalLabels, func(querier storage.Querier, hints *storage.SelectHints, filteredMatchers []*labels.Matcher) error {
 				// The streaming API has to provide the series sorted.
-				set, _, err := querier.Select(true, hints, filteredMatchers...)
-				if err != nil {
-					return err
-				}
+				set := querier.Select(true, hints, filteredMatchers...)
 
 				return remote.StreamChunkedReadResponses(
```

Comment:
This doesn't seem to be handling warnings.

Reply:
Previously, warnings were being ignored. How should we proceed with it in here? Should we log warnings? Should we include the warnings in the response?

Reply:
They shouldn't be even happening currently and remote read doesn't support them, so logging is probably fine.

Reply:
Yea, we talked about this in chunk iterator "in progress" PRs, I think just TODO is fine for now

Reply:
I think back then we said it will be too spammy  :thinking: